ppr-ui/src/components/mhrRegistration/YourHome/AddEditHomeSections.vue

```diff
@@ -8,7 +8,12 @@
     <v-form ref="addEditHomeSectionsForm" v-model="addEditValid">
       <v-row no-gutters>
         <v-col cols="12" sm="2">
-          <label class="generic-label">{{ isNewHomeSection ? 'Add' : 'Edit' }} Section</label>
+          <label
+            class="generic-label"
+            :class="{ 'error-text': validate }"
```

Comment:
It also needs to check if the form is invalid. Only `validate` is not enough. If validation is triggered but the form is valid, the css class will be applied. Please correct me if I'm wrong.

Reply:
This is a unique scenario. Having the form open when returning from the review confirm screen makes the section invalid.

Reply:
I confirmed with Tracey that we showcase the left-side-error-bar and the red label if the panel is left open.

If you examine the code, this is not part that I changed.
The validation for the section is set to false if the editing section form is open.

![image](https://github.com/bcgov/ppr/assets/77707952/b4d2805e-4cb8-4d05-96f4-24827dea0f12)

Reply:
Sounds good, thanks for checking.